src/debug.c

```diff
@@ -475,8 +475,10 @@ void debugCommand(client *c) {
 "    Return the size of different Redis core C structures.",
 "ZIPLIST <key>",
 "    Show low level info about the ziplist encoding of <key>.",
-"QUICKLIST <key>",
-"    Show low level info about the quicklist encoding of <key>.",
+"QUICKLIST <key> <light-enabled>",
+"    Show low level info about the quicklist encoding of <key>."
+"    <light-enabled> is optional, 0 by default, "
+"    1 for enabling light version of the log",
```

Comment:
maybe the term "light" needs more words to be explained?
let's change the code below so that 0 is light (shallow) and 1 is deep (full), or we can consider it an LOD integer
```suggestion
"QUICKLIST <key> [<0|1>]",
"    Show low level info about the quicklist encoding of <key>."
"    The optional argument (0 by default) sets the level of detail",
```